chapters/projectManagement.typ

```diff
@@ -0,0 +1,208 @@
+= Project Management
+// [] Vorgehen (Prozess, Reviews, Workflows, Qualitätssicherung)
+// [x] Projektplan, 
+//    [] Vergleich ursprüngliche Planung, effektive Ausführung
+// [x] Zeiterfassung (Stunden pro Woche/Stunden pro Task-Kategorie, wie Implementation Doku, Meeting, etc.)
+// Hinweis: Keine Liniendiagramme für die Darstellung von Zeit/Arbeitsaufwand pro Woche
+
+This section describes the approach of the project as well as the project plan and time tracking.
+In @approach, it is explained how the project was approached.
+@project-plan, looks into the planning of the project, including a comparative analysis between the initial plan and its effective implementation.
+Additionally, @time-tracking, provides a summarized overview of the allocated working hours coupled with a reflective assessment of the time invested in the reflects on the time invested.
```

Comment:
"with a reflective assessment of the time invested in the reflects on the time invested"
de Satz verstahni ned ganz. Was meinsch do?